hardhat/deployments/localhost/MyERC1155.json

```diff
@@ -494,4 +494,4 @@
       }
```

Comment:
this file can be removed, but then you will need to edit the SymfoniContext, too, same goes for all the other json files below. we really need to remove them, they aren't playing any role in here